os/os.proto

```diff
@@ -204,6 +233,8 @@ message TransferProgress {
 
 // The SyncProgress message signals the Client about the progress of
 // transferring the OS package between Supervisors.
+// Systems which install the package on all Supervisors do not need to
+// transfer the package between Supervisors and will not send SyncProgress.
```

Comment:
A dual Supervisor Target which requires installing the entire system with one Install RPC, does not need to transfer the package between Supervisors and MUST not send SyncProgress.

Reply:
this is missing

Reply:
done.